doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -574,56 +568,38 @@ Navigation in the proof tree
    :n:`@natural:`
      Focuses on the :token:`natural`\-th goal to prove.
 
-   :n:`[ @ident ]: %{`
-     Focuses on the named goal :token:`ident`.
+.. _focus_shelved_goal:
 
-   .. note::
+   :n:`[ @ident ]: %{`
+     Focuses on the goal named :token:`ident` even if the goal is not in focus.
+     Goals are :ref:`existential-variables`, which don't have names by default.
```

Comment:
```suggestion
     Goals are :term:`existential variables <existential variable>`, which don't have names by default.
```

Reply:
Hmm, that suggests you'd favor dropping the "See X" I mention in https://github.com/coq/coq/pull/16498#discussion_r983886282

Reply:
Yes, that's a typical example where I don't think linking to the section is useful.